### PR TITLE
Adds some new emotes and an IPC dialup sound on revival

### PR DESCRIPTION
**Pull request by Bokkiewokkie**
## About The Pull Request

Ports BeeStation/NSV13#1840, but without the felenid emotes.
Also moves the inhale/exhale emotes to mob/emote.dm because they're not /living emotes.

## Why It's Good For The Game

New emotes allow for more interactive RP

## Changelog
:cl: BetaCentari
add: Added hissing, humming and clicking your tongue emotes
add: Added zap emote for ethereals
code: Added unique message option for insect-like mobs (moths, apids and flypeople)
soundadd: Added a dialup sound for IPC players when they are revived
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/IPC.dm [hunk 1]
```diff
@@ -216,6 +216,7 @@
 	H.grab_ghost()
 	H.dna.features["ipc_screen"] = "BSOD"
 	H.update_body()
+	playsound(H, 'nsv13/sound/voice/dialup.ogg', 25)
```
Comment (mcmeiler) [inline code/modules/mob/living/carbon/human/species_types/IPC.dm]:
```suggestion
	playsound(H, 'sound/voice/dialup.ogg', 25)
```

Comment (Bokkiewokkie) [reply code/modules/mob/living/carbon/human/species_types/IPC.dm]: whoops

**Comment by ivanmixo** [comment]
>no purr emote

**Comment by PowerfulBacon** [comment]
Sound source?

**Comment by Bokkiewokkie** [comment]
> Sound source?

Calling a dial-up internet provider? I doubt the respective provider and modem programmer have copyrighted their dial-up sound.

**Comment by Bokkiewokkie** [comment]
> > no purr emote

Last I heard felenids were getting removed from here, so adding it would be pointless.

**Comment by PowerfulBacon** [comment]
Was the sound recorded by you or got off the internet?

**Comment by Pirill** [comment]
Where cat emote :(

**Comment by ivanmixo** [comment]
> > > no purr emote
> 
> Last I heard felenids were getting removed from here, so adding it would be pointless.

FROM WHO LMAO

**Comment by Rukofamicom** [comment]
> Last I heard felenids were getting removed from here

I find this almost as hilarious as the fact you believed it lmao

**Comment by Bokkiewokkie** [comment]
> Was the sound recorded by you or got off the internet?

Taken from freesound.org under CC0, with the original recording being from connecting to NetZero's dial-up service.

**Review by PowerfulBacon — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
good, no purr

**Comment by Fronsis** [comment]
WAIT does this mean that moths will get *chitter ?????? PLEASE i must know @Bokkiewokkie

**Comment by Lagomorphica** [comment]
Bring *purr.

**Comment by mcmeiler** [comment]
Bring *purr . And *nya , perhaps.

**Comment by ivanmixo** [comment]
The people demand purr

**Comment by Fronsis** [comment]
And *chiitter don't Forget about that!

**Comment by PowerfulBacon** [comment]
we are not getting purr

**Comment by Hardly3D** [comment]
> we are not getting purr

1984

**Pull request merged**